extensions/objects/templates/LogicQuestionEditor.js

```diff
@@ -17,20 +17,15 @@
 // in via initArgs.
 
 oppia.directive('logicQuestionEditor', [
-  '$compile', 'OBJECT_EDITOR_URL_PREFIX',
-  function($compile, OBJECT_EDITOR_URL_PREFIX) {
+  'UrlInterpolationService', 'OBJECT_EDITOR_URL_PREFIX',
+  function(UrlInterpolationService, OBJECT_EDITOR_URL_PREFIX) {
```

Comment:
The parameters should be alphabetically ordered.
Ditto for all similar cases below.

Reply:
Hmm, okay I thought constants should be always last.

Reply:
Don't know, sorry. @apb7 -- what is our current convention?

Reply:
Hi @seanlip! We only follow the alphabetic order for injected dependencies currently, and not for parameters in JS files. The rule we follow: [angular/di-order](https://github.com/Gillespie59/eslint-plugin-angular/blob/master/docs/rules/di-order.md).

Reply:
Ok, I will keep it for now.

Reply:
Other than this, the PR looks good.